quiz-template.html

```diff
@@ -1,148 +1,189 @@
-
-/* Different choice attributes */
-var categoryName;
-var description;
-
-/* Hides the quiz page */
-function hideHome(){
-	var div = document.getElementById('home');
-	div.style.visibility = "hidden";
-	div.style.display = "none";
-}	
-
-/* Calculates which category a user fits in */
-function grade(){
-	for(var q=1; q<=totalQuestions;q++){ // all the questions 
-		var currentQuestionName = "question" + q;
-        var currentQuestions = document.getElementsByName(currentQuestionName);
-        for (var c = 0; c < currentQuestions.length; c++) {
-            var radio = currentQuestions[c];
-			if(radio.checked != true){
-                continue;
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html>
+  <head> 
+    <title>Quiz Title</title>
+    <!-- This quiz is meant to be used for personality tests but can be adapted to suit other quiz needs. --> 
+
+    <meta charset="utf-8">
+
+    <style>
+      body {
+        background-color: #FFFFFF;
+        color: #000000;
+        font-family: sans-serif;
+      }
+
+      #result {
+        display: none;
+        text-align: center;
+      }
+
+      #result-category {
+        font-size: larger;
+      }
+
+      fieldset.question {
+        border-width: 0;
+      }
+
+      /* Question headers */
+      fieldset.question legend {
+        font-weight: bold;
+        font-size: larger;
+      }
+    </style>
+
+    <script>
+      /* Global variables*/
+      var totalQuestions = 3;
+      var totalChoices = 3;
+
+      /* Different choice point totals */
+      var xTotal = 0;
+      var yTotal = 0; 
+      var zTotal = 0;
+
+      /* Different choice attributes */
+      var categoryName;
+      var categoryDescription;
+
+      /* Hides the quiz page */
+      function hideMain() {
+        document.getElementById("quiz-main").style.display = "none";
+      }	
+
+      /* Calculates which category a user fits in */
+      function grade() {
+        for(var q = 1; q <= totalQuestions; q++) { // all the questions 
+          var currentQuestionName = "question" + q;
+          var currentQuestions = document.getElementsByName(currentQuestionName);
+
+          for (var c = 0; c < currentQuestions.length; c++) {
+            var option = currentQuestions[c];
+            if (!option.checked) {
+              continue;
             }
-            console.log(radio.value);
-            if(radio.value == "x"){ // category x
-                xTotal += 1;
-            } else if(radio.value == "y"){ // category y
-                yTotal += 1;
-            } else if (radio.value == "z"){ // category z
-                zTotal += 1;
+            // console.log(option.value);

```

Comment:
Comment out or remove debugging statements. Additionally, `console` isn't supported in every browser, and the browser won't use your script if it doesn't support it.